app/src/main/java/com/ds/avare/LocationActivity.java

```diff
@@ -907,41 +908,32 @@ public Object callback(Object o, Object o1) {
                         /*
                          * A/FD
                          */
-                        if (!mAirportPressed.contains("&")) {
-                            mService.setLastAfdAirport(mAirportPressed);
+                        if (mDestinationPressed.getType() == Destination.BASE) {
```

Comment:
having type of a destination pressed is nice here as we select various actions base on type; this also gets rid of checking hard-coded & character